jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -39,24 +42,20 @@ def _viewer_selected_changed(self, event={}):
             return
 
         viewer = self.viewer.selected_obj
+        self.has_show_uncertainty = isinstance(viewer, SpecvizProfileView)
+        if self.has_show_uncertainty:
+            # TODO: refactor so that an API call to viewer.show_uncertainties, which would
+            # set viewer.display_uncertainties would then update the widget state.

```

Comment:
this is a bigger effort and out-of-scope for now.... if we're going this far, it might be just as easy to move the logic upstream into glue-jupyter and handle it all in the viewer state.

Reply:
I wonder... instead of checking `isinstance`, why not do a `hasattr(viewer, 'display_uncertainties')`? Maybe that discussion has happened and I missed it?

Reply:
Okay, let's leave it as is. I don't see how this would work in Imviz anyway. In images, uncertainty is stored as a different image extension (`ERR`) anyway.